### DISCUSSION TITLE
Multi-Site Support

### DISCUSSION THREAD

**Discussion by jackabox** [statamic-rad-pack/shopify#92] [Feature Requests]
It's been requested a few times and although I haven't tested the Shopify add-on with multi-site yet it'd be good to get an idea of what features are wanted from support. This change would probably need to be baked into a v2 of the add-on which involves updates and using GraphQL on the front-end.

- Multi-currency (£/$/etc).
- Different products per site.
- Sync to keep everything in touch.

If anyone else has any ideas/features they'd like to see built into this, then it'd be good to collate these before I begin work on it.

**Comment by DanielDarrenJones** [comment]
This would be ideal for an upcoming project i'm working on, ideally id like a way of importing a different set of products per site, either using a separate isolated Shopify store for each or if possible a way of segmenting these out using one Shopify store. I'm not sure Shopify supports multisite itself however though, so the separate store per site would be easiest I would imagine?

**Reply by jackabox** [reply]
Shopify doesn't support "multisite" but it does support multi-currency/multi-lingual. These can all be processed via one store though, so at the moment it's not a high priority.

It's definitely on my radar, but I believe the work would be quite vast for potentially limited use, especially for something I'm not using myself. If you need it urgently you can drop me a line and we can talk about coding it out.

**Comment by CptCompiler** [comment]
hi @jackabox.
Thank you for this nice plugin. We would need multi currency support for our client. Have you already thought about how to integrate this?

**Reply by jackabox** [reply]
Hey @CptCompiler, thanks! It's not on my radar at the moment. 

With a bit of work, you could update the front-end queries and pass through a currency parameter which would update the value displayed. Also, happy to take a look if you fancied opening a PR with some updates to add multi region pricing.

**Comment by ryanmitchell** [comment]
Going to support translations when this merges https://github.com/statamic-rad-pack/shopify/pull/184

I cant see any information on different currencies and the Admin REST Api - if you can supply a link to the docs on where you see that being returned I can look into it further.

**Reply by DanielDarrenJones** [reply]
If you need set pricing per product the. The general approach is to have a store for each currency separately as Shopify plus only support conversion rate based pricing for multi site.

It would be super helpful to be able to set the Shopify credentials for each site so we can have a separate store for different languages

**Reply by ryanmitchell** [reply]
Not sure if/how that could work, as the action urls are not site specific?
